packages/frontend-app-api/api-report.md

```diff
@@ -12,7 +12,8 @@ import { JSX as JSX_2 } from 'react';
 // @public (undocumented)
 export function createApp(options: {
   plugins: BackstagePlugin[];
-  config?: ConfigApi;
+  configLoader?: () => Promise<ConfigApi>;
+  pluginLoader?: (ctx: { config: ConfigApi }) => Promise<BackstagePlugin[]>;
```

Comment:
kinda another discovery?

Reply:
yeah, dynamic loading. Just something simple for now to enable experimentation, can figure out details later :>